### DISCUSSION TITLE
C operators

### DISCUSSION THREAD

**Discussion by 11django11** [11django11/test-project#10] [Q&A] [answered]
share some C operators pls

**Comment by seldish-og** [comment, marked as answer]
+ | Adds two operands. | A + B = 30
-- | -- | --
− | Subtracts second operand from the first. | A − B = -10
* | Multiplies both operands. | A * B = 200
/ | Divides numerator by de-numerator. | B / A = 2
% | Modulus Operator and remainder of after an integer division. | B % A = 0
++ | Increment operator increases the integer value by one. | A++ = 11
-- | Decrement operator decreases the integer value by one. | A-- = 9


<h2></h2>
